src/WebJobs.Script/Binding/DocumentDBBinding.cs

```diff
@@ -33,6 +34,8 @@ internal class DocumentDBBinding : FunctionBinding
 
         public bool CreateIfNotExists { get; private set; }
 
+        public string ConnectionString { get; private set; }
```

Comment:
I think your actual attribute uses the name "ConnectionString" even though it is really an app setting name, not a connection string. I know the xml doc explains things, but we should probably rename these consistently to something like "Connection".